src/test/unit_tests/test_nc_cli.js

```diff
@@ -991,16 +991,25 @@ mocha.describe('manage_nsfs cli', function() {
             assert_whitelist(config_data, config_options);
         });
 
-        mocha.it('cli whitelist ips is empty', async function() {
+        mocha.it('should fail - cli whitelist ips is empty', async function() {
             try {
                 await exec_manage_cli(type, '', { config_root, ips: '' });
                 assert.fail('should have failed with whitelist ips should not be empty.');
             } catch (err) {
-                assert_error(err, ManageCLIError.InvalidWhiteListIPFormat);
+                assert_error(err, ManageCLIError.UnsetArgumentIsInvalid);
```

Comment:
I think the error we're throwing here is confusing `UnsetArgumentIsInvalid` ("Argument can not be unset, its value must be set or check there are no leading spaces") vs `InvalidWhiteListIPFormat` ("Whitelist IP body format is invalid") that we used to have, as the option `--ips` can be unset using `[]` and not `''`.

Reply:
sure, do you have a better suggestion?

Reply:
I think the simple one will be to add the `ips` to the `LIST_UNSETTABLE_OPTIONS` (and you can add a comment there), and we will end with the same error that we had before.

Reply:
@shirady I eventually thought that we need to still throw the same error code but I added more details and a resolution
> noobaa-cli whitelist --ips ''
{
  "error": {
    "code": "UnsetArgumentIsInvalid",
    "message": "Argument can not be unset or it was unset incorrectly, its value must be set correctly or check there are no leading spaces",
    "detail": "flag value of ips is '' but this option can't be unset via ''. Please use '[]' instead."
  }
}